Neos.Fusion/Classes/FusionObjects/ComponentImplementation.php

```diff
@@ -51,37 +52,32 @@ public function evaluate()
      * @param array $context
      * @return array
      */
-    protected function prepare($context)
+    protected function prepare(array $context): array
```

Comment:
Q: Is this supposed to be extended/inherited and hence the added typehints to be considered breaking?

Reply:
This is not @api and no intended extension point. We only seperated this into a special method so we can address this in aspects seperately.

So for me this is ok.